### DISCUSSION TITLE
2006/08/09/labview-%E4%B8%AD%E7%9A%84%E6%95%B0%E5%AD%97%E5%9E%8B%E6%95%B0%E6%8D%AE-3-%E6%95%B0%E5%80%BC%E7%9A%84%E5%8D%95%E4%BD%8D

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#105] [Announcements]
# 2006/08/09/labview-%E4%B8%AD%E7%9A%84%E6%95%B0%E5%AD%97%E5%9E%8B%E6%95%B0%E6%8D%AE-3-%E6%95%B0%E5%80%BC%E7%9A%84%E5%8D%95%E4%BD%8D

1\. 数值控件上的单位

https://blog.qizhen.xyz/2006/08/09/labview-%E4%B8%AD%E7%9A%84%E6%95%B0%E5%AD%97%E5%9E%8B%E6%95%B0%E6%8D%AE-3-%E6%95%B0%E5%80%BC%E7%9A%84%E5%8D%95%E4%BD%8D

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Gavin
2007年10月14日 21:30 编辑
写得好，支持

回复
军临天下
2006年11月1日 16:24 编辑
 老兄，继续啊！

回复
binglan
2006年08月10日 23:22 编辑
找到你很高兴，别害怕，我们不认识。只是有很多问题请教你！

回复
永青
2006年08月10日 09:40 编辑
unit label有点意思的
